the-collab-lab/tcl-28-smart-shopping-list
# Ks remove footer

## Description


- Removed Footer component
- Removed inclusion of Footer component in App.js
- Removed footer-relating styling in index.css

## Related Issue


closes #46 

## Acceptance Criteria


- [x] Remove Footer component
- [x] Remove references to Footer component within application

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
| ✓ | :hammer: Refactoring       |

## Updates

### Before


<img width="687" alt="Screen Shot 2021-08-31 at 7 46 22 AM" src="https://user-images.githubusercontent.com/40218057/131513887-f911c40d-7c20-4bf9-a76b-7937dd33b493.png">

### After


<img width="687" alt="Screen Shot 2021-08-31 at 7 43 02 AM" src="https://user-images.githubusercontent.com/40218057/131513354-3ab713dd-b242-4cfa-b939-8f9e3c51da21.png">

## Testing Steps / QA Criteria


- checkout branch `ks-remove-footer`
- navigate to various parts of the app using the browser navigation (`/`, `/list`, `/add`)
- you should still be able to navigate, but not see any links at the bottom of the app
